easybuild/easyconfigs/c/CheckM/CheckM-1.1.3-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,65 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'CheckM'
+version = '1.1.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Ecogenomics/CheckM'
+description = """CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
+ single cells, or metagenomes."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+source_urls = [
+    'https://pypi.python.org/packages/source/c/checkm-genome',
+    'https://data.ace.uq.edu.au/public/CheckM_databases/',
+]
+sources = [
+    'checkm-genome-%(version)s.tar.gz',
+    {
+        'filename': 'checkm_data_2015_01_16.tar.gz',
+        'extract_cmd': "mkdir -p %(builddir)s/data && cd %(builddir)s/data && tar xfvz %s",
+    },
+]
+checksums = [
+    '7fda369783d3b7d6a7324ee33cc93ac3560fba509bf2139e920fa661e2b45644',  # checkm-genome-1.1.3.tar.gz
+    '971ec469348bd6c3d9eb96142f567f12443310fa06c1892643940f35f86ac92c',  # checkm_data_2015_01_16.tar.gz
+]
+
+# Dependencies:
+# https://github.com/Ecogenomics/CheckM/blob/master/setup.py
+dependencies = [
+    ('Python', '3.8.2'),
+    ('pplacer', '1.1.alpha19', '', True),
+    ('prodigal', '2.6.3'),
+    ('HMMER', '3.3.1'),
+    ('SciPy-bundle', '2020.03', versionsuffix),  # numpy
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('Pysam', '0.16.0.1'),
+    ('DendroPy', '4.4.0'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+# also install CheckM databases, see https://github.com/Ecogenomics/CheckM/wiki/Installation#how-to-install-checkm
+postinstallcmds = [
+    "cp -a %(builddir)s/data %(installdir)s",
+    "PYTHONPATH=%(installdir)s/lib/python%(pyshortver)s/site-packages:$PYTHONPATH "
+    "%(installdir)s/bin/checkm data setRoot %(installdir)s/data",
+]
+
+sanity_check_paths = {
+    'files': ['bin/checkm'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_commands = ["checkm test %(builddir)s/checkm_test_results"]
+
+sanity_pip_check = True
+
+moduleclass = 'tools'
```

Comment:
@Darkless012 oughtn't this to be `bio`?

Reply:
All past easyconfigs are `tools`. Makes sence to move it to bio though?

Reply:
Maybe merge and then update all .eb to same `moduleclass`?

Reply:
@Darkless012 It looks pretty unambiguously like `bio` to me. Personally I would just change it for this update and leave the old ones as they are. @boegel ?

Reply:
Change it to `bio` here, and open a *separate* PR to fix the `moduleclass` in the existing `CheckM` easyconfigs to `bio`.

Reply:
ack

Reply:
moduleclass fixed in #11662